From 6ba2b98c2f7813862aeecb4d12e3ba8108db8457 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Nils=20H=C3=B6lscher?= <nilhoel1@gmail.com>
Date: Tue, 11 Jan 2022 13:10:17 +0100
Subject: [PATCH] build with Gurobi and Cplex

---
 CMakeLists.txt         | 2 +-
 include/Util/Options.h | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9befd39..216d519 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,5 +30,5 @@ add_llvm_tool(llvmta
 
 add_subdirectory(lib)
 
-target_link_libraries(llvmta PRIVATE lpsolve55 llvmtaLLVMPasses llvmtaUtil llvmtaPathAnalysis llvmtaAnalysisFramework llvmtaPartitionUtil llvmtaPreprocessing llvmtaMemory llvmtaMicroarchitecture llvmtaMicroarchitectureOutOfOrder llvmtacrpd)
+target_link_libraries(llvmta PRIVATE lpsolve55 gurobi cplex llvmtaLLVMPasses llvmtaUtil llvmtaPathAnalysis llvmtaAnalysisFramework llvmtaPartitionUtil llvmtaPreprocessing llvmtaMemory llvmtaMicroarchitecture llvmtaMicroarchitectureOutOfOrder llvmtacrpd)
 
diff --git a/include/Util/Options.h b/include/Util/Options.h
index 49cf0e5..d737008 100644
--- a/include/Util/Options.h
+++ b/include/Util/Options.h
@@ -27,8 +27,8 @@
 #ifndef UTIL_OPTIONS_H
 #define UTIL_OPTIONS_H
 
-//#define CPLEXINSTALLED
-//#define GUROBIINSTALLED
+#define CPLEXINSTALLED
+#define GUROBIINSTALLED
 
 #include "llvm/Support/CommandLine.h"
 
-- 
2.34.1

